src/Manager.php

```diff
@@ -124,20 +124,34 @@ public function createReleaseBranch($parent, $remote, $version)
         $escaped_parent = escapeshellarg($parent);
         $escaped_release = escapeshellarg($release);
 
-        `git fetch -q $escaped_remote $escaped_parent`;
-
-        $command = sprintf(
-            'git checkout -q -b %s %s/%s',
-            $escaped_release,
+        // Fetch only the parent branch from the remote.
+        $fetch_command = sprintf(
+            'git fetch -q %1$s %2$s:refs/remotes/%1$s/%2$s',
```

Comment:
Nope. That fetches the remote branch into a local branch. The above change fetches the remote branch into the local remote branch.